src/Bookings.js

```diff
@@ -1,21 +1,52 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+//import "./Bookings.css";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
 
 const Bookings = () => {
+  const [bookings, setBookings] = useState(null);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(null);
+  useEffect(() => {
+    fetch("https://cyf-react.glitch.me")
+      //    fetch("https://cyf-react.glitch.me/delayed")
+      //fetch("https://cyf-react.glitch.me/error")
+      .then(response => response.json())
+      .then(data => {
+        if (data.error) {
+          throw data;
+        }
+        console.log(data);
+        setBookings(data);
+        setLoading(false);
+      })
+      .catch(error => setError(error));
+    console.log("500 HTTP error");
+  }, []);
   const search = searchVal => {
     console.info("TO DO!", searchVal);
+    let filteredBooking = bookings.filter(
+      booking =>
+        booking.firstName.toLowerCase().includes(searchVal.toLowerCase()) ||
+        booking.surname.toLowerCase().includes(searchVal.toLowerCase())
+    );
+    setBookings(filteredBooking);
   };
-
-  return (
-    <div className="App-content">
-      <div className="container">
-        <Search search={search} />
-        {/* <SearchResults results={FakeBookings} /> */}
+  if (loading && !error) {
```

Comment:
You could make this slightly more readable with a ternary operator:
error ? (
...
) : isLoading ? (
...
) : (
...
)